@here/harp-omv-datasource/lib/OmvDataSource.ts

```diff
@@ -182,7 +182,10 @@ export class OmvDataSource extends TileDataSource<OmvTile> {
             concurrentDecoderServiceName: OMV_TILE_DECODER_SERVICE_TYPE,
             decoder: m_params.decoder,
             concurrentDecoderScriptUrl: m_params.concurrentDecoderScriptUrl,
-            copyrightInfo: m_params.copyrightInfo
+            copyrightInfo: m_params.copyrightInfo,
+            minZoomLevel: m_params.minZoomLevel || 1,
```

Comment:
There is a hidden "truthy" in it. Either make it work without it, or use "getOptionValue"

Reply:
Good catch.